webapp/src/components/markdownEditor.test.tsx

```diff
@@ -14,6 +14,7 @@ import {MarkdownEditor} from './markdownEditor'
 jest.mock('../utils')
 jest.useFakeTimers()
 jest.mock('draft-js/lib/generateRandomKey', () => () => '123')
+jest.setTimeout(30000)
```

Comment:
A test in `markdownEditor.test.tsx` was taking too long to complete and thus generated error. That's why I put a a timeout to increase the default timeout of 5000ms.

Reply:
It's strange, it worked for me in my local. Anyways it doesn't matter that much. Let's stick to the default timeout offered by jest to avoid unnecessary changes.